packages/x-archetype-utils/jest.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  moduleFileExtensions: ['ts', 'js'],
+  transform: {
```

Comment:
Is there a real difference between using the `preset: 'ts-jest'` or using transform for just *.ts files?

Should we change the `testEnvironment` to `jsdom`?

Reply:
As javi said, use the preset